pkgs/development/python-modules/xmlsec/default.nix

```diff
@@ -0,0 +1,36 @@
+{ pkgs, lib, stdenv, fetchPypi, buildPythonPackage, pytestCheckHook,
+pkgconfig, setuptools_scm, toml, lxml, hypothesis }:
+
+buildPythonPackage rec {
+  pname = "xmlsec";
+  version = "1.3.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1c4k42zv3plm6v65p7z6l5rxyf50h40d02nhc16vq7cjrfvdf4rx";
+  };
+
+  # https://github.com/mehcode/python-xmlsec/issues/84#issuecomment-632930116
+  patches = [
+    ./reset-lxml-in-tests.patch
+  ];
+
+  buildInputs = [
+    pkgs.xmlsec pkgs.libxslt pkgs.libxml2 pkgs.libtool
```

Comment:
Please inherit them from python.nix.

Reply:
Sorry, could you elaborate on what you mean by that?

Reply:
in ``pkgs/top-level/python-packages.nix``:
```nix
wxPython30 = callPackage ../development/python-modules/wxPython/3.0.nix {
  inherit (pkgs) libtool libxml2 pkg-config xmlsec;
};
```